src/js/utils/math.util.js

```diff
@@ -77,6 +78,15 @@ export class MathUtil {
       .toFixed(DECIMAL_PLACES)
   }
 
+  static percent (a, percent, ROUND_TYPE = ROUNDING_MODES.ROUND_UP) {
```

Comment:
percent is not so descriptive. also `a` and `percent` aren't good. mb `value` and `ratio`? up to u